IaC/BareMetal/loderunner/baseline-memory.yaml

```diff
@@ -33,4 +33,4 @@ spec:
           key: Region
 
     - name: PodType
-      value: webv
```

Comment:
@aflinchb - we should standardize the PotType for reporting

Do we want to tag baseline tests the same as benchmark tests?

They shouldn't go to the same Log Analytics, but, they could. Should we tag them with "dev" to avoid bad data?

Reply:
When you say tag do you mean the PodType name? Or is there another way it would be tagged? I like tagging them with dev or test to distinguish easily in logs (even if they don't go to the same Log Analytics).   I think it's worth distinguishing between baseline and benchmark just in case there is some difference in behavior.

Reply:
yes, pod type name - we discussed in CR